packages/electron-mksnapshot/README.md

```diff
@@ -0,0 +1,27 @@
+## @packages/electron-mksnapshot
+
+A rewrite of [electron/mksnapshot](https://github.com/electron/mksnapshot) to support multiple
+versions.
+
+The main difference is that the _mksnapshot_ binary is not downloaded when installing this
+module. 
+
+Instead whenever it is run an electron version it should make a snapshot for can be
+provided or is resolved from the electron installed relative to the root of your project.
```

Comment:
I can tell you that this sentence needs some punctuation, but I don't think I can tell you exactly where 🤔

Reply:
Updated it. Let me know if that makes it clearer